frontend/src/components/TextInput.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import styled from 'styled-components'
```

Comment:
Another tip! Use 'styled-components/macro' to make is easier to read in dev tools